BrainUp/BrainUpTests/Extensions/UIFontTestCase.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  UIFontTestCase.swift
+//  BrainUpTests
+//
+//  Created by Evgenii Zhigunov on 3/18/22.
+//
+
+import XCTest
+@testable import BrainUp
+
+class UIFontTestCase: XCTestCase {
+    private let testFontSize: CGFloat = 12
+
+    func testMontserratRegular() {
+        let fontName = "Montserrat-Regular"
+        guard let testFont = UIFont(name: fontName, size: testFontSize) else {
+            XCTFail("Font \(fontName) not found")
+            return
+        }
+        XCTAssertEqual(testFont, UIFont.montserratRegular(size: testFontSize))
+    }
+    
+    func testMontserratBold() {
+        let fontName = "Montserrat-Regular"
```

Comment:
The same as for colors. And just in case, I left this function (for bold and semibold fonts) for future, I  added fonts files in my branch. And you test here that MontessratBold return MontessratRegular font.